src/Akka.Persistence.Sql.Hosting/SqlJournalOptions.cs

```diff
@@ -154,32 +154,41 @@ protected override StringBuilder Build(StringBuilder sb)
 
             base.Build(sb);
 
-            if (IsDefaultPlugin)
-            {
-                sb.AppendLine("akka.persistence.query.journal.sql {");
-                sb.AppendLine($"connection-string = {ConnectionString.ToHocon()}");
-                sb.AppendLine($"provider-name = {ProviderName.ToHocon()}");
+            BuildQueryConfig(sb, QueryPluginId);
+            
+            if (IsDefaultPlugin && Identifier is not "sql")
+                BuildQueryConfig(sb, "akka.persistence.query.journal.sql");
+
+            return sb;
```

Comment:
The actual fix.
Before, we included the default fallback but did not do any overrides, that's why the connection string and provider name was empty.
The first `BuildQueryConfig()` line injects all that information.

Reply:
LGTM